common/lib/xmodule/xmodule/x_module.py

```diff
@@ -751,7 +750,7 @@ def __delete__(self, instance):
 
 
 @XBlock.needs("i18n")
-class XModule(HTMLSnippet, XModuleMixin):  # pylint: disable=abstract-method
```

Comment:
abstract-method is not disabled, there are still a couple dozen suppressions in the code.  I don't understand all the logic, but the recent Pylint changes (http://docs.pylint.org/changelog.html) include a few mentions of improving the smarts around abstract methods.